dist/package.sh

```diff
@@ -28,7 +28,12 @@ if [ -e "release" ]; then
     done
 fi
 
-VERSION=$(grep "\"version\"" package.json  | sed 's/\s*\"version\": \"\([^\"]*\).*/\1/')
+if [[ "$OSTYPE" == "darwin"* ]]; then
+        VERSION=$(grep "\"version\"" package.json  | sed 's/[[:blank:]]*\"version\": \"\([^\"]*\).*/\1/')
+    else
+       VERSION=$(grep "\"version\"" package.json  | sed 's/\s*\"version\": \"\([^\"]*\).*/\1/')
+    fi
```

Comment:
Sorry if my last comment was confusing. You can use `[[:blank:]]` for both darwin and non-darwin, so the `if` can be removed altogether.

Reply:
I was just about to try it and fixed it in https://github.com/threema-ch/threema-web/pull/406/commits/95883b815973c4c91681315260d25d3ddbea1d5a